solidity/random-beacon/test/RandomBeacon.test.ts

```diff
@@ -0,0 +1,213 @@
+import { ethers } from "hardhat"
+import { Signer, Contract } from "ethers"
+import { expect } from "chai"
+
+describe("RandomBeacon", () => {
+  let governance: Signer
+  let thirdParty: Signer
+  let randomBeacon: Contract
+
+  beforeEach(async () => {
+    const signers = await ethers.getSigners()
+    governance = signers[0]
+    thirdParty = signers[1]
+
+    const RandomBeacon = await ethers.getContractFactory("RandomBeacon")
+    randomBeacon = await RandomBeacon.deploy()
+    await randomBeacon.deployed()
+  })
+
+  describe("updateRelayEntryParameters", () => {
+    const RELAY_REQUEST_FEE = 100
```

Comment:
We were usually naming this kind of constants in camelCase. Is linter complaining about it?

Reply:
No, I could correct it without problems in https://github.com/keep-network/keep-core/pull/2613/commits/082c206fef245c4b2095effd8eefce40622a5b25.